.github/workflows/simple.yml

```diff
@@ -13,3 +13,6 @@ jobs:
       run: |
         python -c "print(3+2)"
         python --version
+    - name: Get running processes for the current user
+      run: |
+        ps -u $USER
```

Comment:
Cool! Can you add `df -h` as well?

Reply:
Build fails on my fork, let me debug it.